### PR TITLE
move initialization services to fileserver

### PR DESCRIPTION
**Pull request by jialin-li**

### CODE REVIEW

File: installer/fileserver/server.go [hunk 1]
```diff
@@ -81,17 +100,82 @@ func Init(conf *config) {
...
 	t.Certificates = []tls.Certificate{c.cert}
 	s := &http.Server{
 		Addr:      c.addr,
-		Handler:   http.FileServer(http.Dir(c.serveDir)),
+		Handler:   mux,
 		TLSConfig: t,
 	}
 
-	log.Infof("Starting server on %s", s.Addr)
+	log.Infof("Starting installer-engine server on %s", s.Addr)
```
Comment (morris-jason) [inline installer/fileserver/server.go]: Fileserver?

Comment (morris-jason) [inline installer/fileserver/server.go]: Fileserver?

File: installer/fileserver/html/exec.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        {{ end }}
+
+    </main>
+</div>
+</body>
+</html>
+
+
+
+
```
Comment (morris-jason) [inline installer/fileserver/html/exec.html]: Not a fan of calling this file exec.html.

File: installer/fileserver/server.go (hunk 1 above)
Comment (jialin-li) [inline installer/fileserver/server.go]: Oops, fixed!

**Pull request merged**